### ISSUE TITLE
Markdown not showing? 🤔 

### ISSUE THREAD

**Issue by jinhanfromNUS** [nus-cs2103-AY2324S2/forum#495]
I have found some issues while reviewing my peer's user guide. It seems that in the website (github.io), it is unable to present the [!NOTE], [!WARNING], ... However, in the github itself (while reading it directly from README.md), the [!NOTE], [!WARNING], is presented well.

I found this kind of issue while reviewing other people's work, and then recheck my own to find out that this issue is also presented on my own user guide too. Is this some kind of bug 🤔 ?

Below are the images for further reference.
1st image: from github.io
2nd image: from README.md

![Screenshot 2024-03-10 001541](https://github.com/nus-cs2103-AY2324S2/forum/assets/111408810/6f2fe51f-fc8b-48a1-b9ca-5578eb387ba8)
![Screenshot 2024-03-10 001615](https://github.com/nus-cs2103-AY2324S2/forum/assets/111408810/9b7cb711-d2dd-46c8-a906-90f04975b84d)

**Comment by damithc** [comment]
@jinhanfromNUS let's wait to see what others think ...

**Comment by vnnamng** [comment]
[issue](https://github.com/orgs/community/discussions/77077) similar issue from Nov 2023
I had the same issue last time and it is still not fixed. 
Tldr: different markdown renderer for GitHub repo preview and GitHub page.
Afaik, many emotes from GitHub markdown also do not work on GitHub page.
What that person can do is to use the tradition method of doing alert (tables etc...).

**Comment by jinhanfromNUS** [comment]
I have found out a way to ensure the availability of emotes. 

I have also encountered similar issues on emojis too (emojis not showing, just like above). I copied the emojis online and pasted it into the README.md, but it is not shown at all in the website!

The way I resolve this is by typing `:` into the markdown, and the list of available emojis will appear. As far as I concern, I still have not encountered any single case where the emojis disappear when using the method above.

Another thing worth noting is, even when the emoji is available on github, for example: 🖼️ (\:framed_picture), it might not appear on your website if you copied and pasted from online.
You can try this on your markdown
Using the `:` method: 🖼️
Using copy and paste from online: 🖼️ (Source: https://emojipedia.org/framed-picture)

Nevertheless, the issue of quoting is yet to resolved :(
